docs/source/main_concepts/adapters.rst

```diff
@@ -121,37 +311,245 @@ Meta-json sink adapter writes received messages as newline-delimited JSON stream
 - Local path to a single file
 - Local path with substitution patterns:
   - ``%source_id`` will insert ``SOURCE_ID`` value into resulting filename
-  - ``%src_filename`` will insert source filename into resulting filename
+  - ``%src_filename`` will insert source filename into resulting filename. This option is available only when using the standard source-adapters or adding the necessary filename information to the custom adapter.
```

Comment:
`This option is available only when using the standard source-adapters or adding the necessary filename information to the custom adapter.` - What does it mean? I can't understand.